app/decorators/sale_decorator.rb

```diff
@@ -112,4 +112,7 @@ def user_email
     object.user.email
   end
 
+  def report_delivery
+    "report_delivery"
```

Comment:
It is used in the view _action_buttons.html.haml
Translated in the view shared/model_state_change